src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx

```diff
@@ -240,6 +246,25 @@ export const EVMDestinationAdaptorProvider = ({
     fallback,
   ]);
 
+  useEffect(() => {
+    if (transactionStatus === "Transfer Completed") {
+      if (!destinationBridge || transferTxHash) return;
+      getTransferTxHashByNonce(
+        destinationChainConfig as EvmBridgeConfig,
+        parseInt(depositNonce as string)
+      ).then((txHash: string) => {
```

Comment:
Let's use `await/async` instead of nesting.

Reply:
The whole code has `then`-style in `useEffect`. Would like to remain the style same everywhere.